core/backend/src/Element.ts

```diff
@@ -336,38 +336,55 @@ export class Element extends Entity {
     return val;
   }
 
-  /** Collect the Ids of this element's *predecessors* at this level of the class hierarchy.
-   * A *predecessor* is an element that had to be inserted before this element could have been inserted.
-   * This is important for cloning operations but can be useful in other situations as well.
-   * @param predecessorIds The Id64Set to populate with predecessor Ids.
-   * @note In order to clone/transform an element, all predecessor elements must have been previously cloned and remapped within the [IModelCloneContext]($backend).
-   * @note This should be overridden (with `super` called) at each level the class hierarchy that introduces predecessors.
-   * @see getPredecessorIds
+  /** Collect the Ids of this element's *references* at this level of the class hierarchy.
+   * @deprecated use collectReferenceIds instead, the use of the term *predecessors* was confusing and became inaccurate when the transformer could handle cycles
```

Comment:
Can you use doc links when recommending alternatives for deprecated APIs (like [here](https://www.itwinjs.org/reference/appui-layout-react/toolbar/toolbarpanelalignment/))?

Reply:
done for all 3 deprecations I made, please check